indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -225,9 +225,15 @@ def reject(self, reason):
         self.booking.reject(session.user, reason)
 
     @use_kwargs({
-        'reason': fields.String(required=False)
+        'reason': fields.String(required=False),
+        'force': fields.Bool(required=False)
     })
-    def accept(self, reason=None):
+    def accept(self, reason=None, force=False):
+        if not force:
+            collisions = get_prebooking_collisions(self.booking)
+            collision_data = reservation_occurrences_schema.dump(collisions)
+            if collision_data:
+                raise ExpectedError('Pre-booking collision', data=collision_data)
```

Comment:
Since this error is never shown to users I wonder if it wouldn't be better to use a more machine-targetted message, like `'prebooking_collision'` (and then also check for it in the JS code instead of just checking for *an* error)